aetherpy/tests/test_io_read.py

```diff
@@ -236,3 +237,80 @@ def test_read_aether_file_dict(self, fname, fvars):
             assert isinstance(data['long_name'][i], str)
 
         return
+
+    @pytest.mark.parametrize('fname', ['3DALL_20110320_003000.nc'])
+    @pytest.mark.parametrize('fvars', [None, ['O']])
+    def test_read_blocked_netcdf_header(self, fname, fvars):
+        """Test successful block-based Aether NetCDF header
+
+        Parameters
+        ----------
+        fname : str
+            File base name
+        fvars : NoneType or list
+            List of variable names to read in or NoneType to read all
+
+        """
+        filename = os.path.join(self.test_dir, fname)
+        assert os.path.isfile(filename), "missing test file: {:}".format(
+            filename)
+
+        self.header = read_routines.read_blocked_netcdf_header(
+            filename)
+        self.eval_header(file_list=False)
+        return
+
+    def test_read_blocked_netcdf_header_bad_file(self):
+        """Test raises IOError with bad filename."""
+
+        with pytest.raises(IOError) as verr:
+            read_routines.read_blocked_netcdf_header("not_a_file")
+
+        assert str(verr).find("unknown aether netCDF blocked file") >= 0
+        return
+
+    @pytest.mark.parametrize('fname', ['3DALL_20110320_003000.nc'])
+    @pytest.mark.parametrize('fvars', [None, ['O']])
+    def test_read_blocked_netcdf_file(self, fname, fvars):
+        """Test successful block-based Aether NetCDF data reading into dict.
+
+        Parameters
+        ----------
+        fname : str
+            File base name
+        fvars : NoneType or list
+            List of variable names to read in or NoneType to read all
+
+        """
+        filename = os.path.join(self.test_dir, fname)
+        assert os.path.isfile(filename), "missing test file: {:}".format(
+            filename)
+
+        data = read_routines.read_blocked_netcdf_file(filename, file_vars=fvars)
+
+        # Evaluate the output keys
+        # NOTE: 'units' and 'long_name' are here for backwards compatibility
```

Comment:
Add an issue for this and replace 'NOTE' with 'TODO(#XXX)' where XXX is the issue number.  This makes the section of the code needed to address an issue easy to find using grep.